tailwind.config.js

```diff
@@ -18,9 +16,23 @@ module.exports = {
     theme: {
         extend: {
             colors: {
-                primary: colors.green[700],
-                secondary: colors.gray[400],
-            }
+                primary: 'rgb(var(--primary)  / <alpha-value>)', // Text color
+                secondary: 'rgb(var(--secondary) / <alpha-value>)', // Text inactive color
+                accent: 'rgb(var(--accent) / <alpha-value>)', // Theme color
+                enhanced: 'rgb(var(--enhanced) / <alpha-value>)', // Checkout conversion color
+                highlight: 'rgb(var(--highlight) / <alpha-value>)' // Background highlight color
+            },
+            borderColor: {
+                DEFAULT: 'rgb(var(--border) / <alpha-value>)',
+                border: 'rgb(var(--border) / <alpha-value>)'
+            },
```

Comment:
We're using this because you can easily adjust the colors for multistore sites. Also adding the alpha value because in tailwind you can add a class with 'bg-primary/50' or 'bg-primary bg-opacity-50'